inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,33 @@
+package inheritance;
+
+import java.util.LinkedList;
+
+public class Restaurant {
+    String name;
+    int rating;
+    String price;
+    int numberofReviews;
+
+    LinkedList<Review>restaurantReviews;
+
+    Restaurant (String name, int rating, String price) {
+        this.name = name;
+        this.rating = rating;
+        this.price = price;
+        this.numberofReviews = 1;
+        this.restaurantReviews = new LinkedList<Review>();

```

Comment:
No need to put the Review type in this LinkedList, since it's already established on Ln11.